src/chainstate/burn/operations/leader_block_commit.rs

```diff
@@ -453,6 +455,14 @@ impl LeaderBlockCommitOp {
             //   we want to make sure that any TX that forces us to perform the check
             //   has either burned BTC or sent BTC to the PoX recipients
 
+            // if we're in the prepare phase, then this block-commit _must_ burn.
+            if burnchain.is_in_prepare_phase(self.block_height) {
```

Comment:
Should we treat the prepare phase block commits the same as we do post-sunset commits? (i.e., single burn output, rather than 2)

Reply:
I'm not sure.  Minimally, we only need to care that we're burning, and that we can measure the burn amount.  A miner ought to be able to create But per #2094, maybe it makes sense to avoid compelling miners to use a particular format?

If anything in this method needs to be more explicit and rigid, I think it should be avoiding a PoX descent check altogether if it's in the prepare phase (will fix).

Reply:
That's fine with me -- the rationale for requiring exactly two commit outputs during the reward cycles is that it levels the playing field between PoX'ers and PoB'ers -- both are required to have the same tx size (modulo the inputs they choose). During the prepare phase, that rationale doesn't apply any more: everyone is burning. So in theory, we could allow a single commit output. But we can also delay this change to a separate PR.

Reply:
I went ahead and made it so a prepare-phase block-commit must have a single burn output